Dockerfile

```diff
@@ -1,26 +1,10 @@
-FROM openjdk:8
-
-ENV SCALA_VERSION=2.12.4 \
-    SBT_VERSION=1.0.4
-
-# Install Scala.
-RUN \
-  curl -fsL https://downloads.typesafe.com/scala/$SCALA_VERSION/scala-$SCALA_VERSION.tgz | tar xfz - -C /root/ && \
-  echo >> /root/.bashrc && \
-  echo "export PATH=~/scala-$SCALA_VERSION/bin:$PATH" >> /root/.bashrc
-
-# Install sbt.
-RUN \
-  curl -L -o sbt-$SBT_VERSION.deb https://dl.bintray.com/sbt/debian/sbt-$SBT_VERSION.deb && \
-  dpkg -i sbt-$SBT_VERSION.deb && \
-  rm sbt-$SBT_VERSION.deb
+FROM mozilla/sbt:8u171_1.1.6
 
+# Install python and moto for spinning up a mock S3 service
```

Comment:
Would bundling it in our image make sense?

Reply:
It might make sense to refactor this to have separate images for running moto and for running sbt. I'd prefer not to add this directly into our published sbt image so that can stay reasonably small.